### PR TITLE
Pair programming exercise contact form

### PR DESCRIPTION
**Pull request by Drew246**
This update includes:

update index.html

update styles.css

Added a contact form

### CODE REVIEW

**Review by Cabraham1 — COMMENTED** [review]
# INVALID ❌ 

Hi @Drew246  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 😓

## Invalid Code Review Request ❌

- [ ] Unfortunately, you have submitted a project that is a pair programming but has no commit from your coding partner please reach out



You can take a step back and read the [lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) for more clarity 💡

_**Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready**_ 🤝🙏

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -174,95 +182,90 @@ <h3>
...
         </div>
         <form action="https://formspree.io/f/mknyobde" method="post">
           <input
             type="text"
             name="name"
             id="name"
-            minlength="30"
-            placeholder="Lorem Ipsum"
+            max-length="30"
+            placeholder="Your name (first and last)"
```
Comment (Ibizugbe) [inline index.html:254]: Nice job integrating the `formspre` service in your form😉. Let's try to make our project match the given design😉. Kindly take note of the spacings (margin and padding) around each element and try to make it match the design😉.


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|image|image|

Also, Kindly ensure the background image is completely present in the contact form  and add some paddings to your button to make it match the design😉.


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|image|image|

File: styles.css [hunk 2]
```diff
@@ -303,10 +303,17 @@ input[type="submit"] {
...
+  background-color: #2230d2;
+  box-shadow: none;
+  color: black;
+}
+
 button:hover,
 input[type="submit"]:hover {
   background-color: #6070ff;
-  color: white;
+  color: #fff;
```
Comment (Ibizugbe) [inline styles.css]: Your button hover state doesn't match the design, Kindly implement the hover to match the design. currently, both the `hover` and `active` states share the same color😉 and this doesn't match the design😉.

image

when hovered
image

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @Drew246 👋👋, 

tryagain


Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights✅✅

- [ ] Linters are passing👍.
- [ ] Formspre service has been integrated to the project👍.
- [ ] The form section is wrapped in a `form` tag👍.
- [ ] Required validations and type attribute has been added👍.


 ## Required Changes ♻️

- [ ] Nice job so far Kindly ensure the layout of your form is done using flexbox as this is a requirement in this milestone👍.
image

image


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @Drew246 👋👋, 

tryagain



Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights✅✅

- [x] Linters are passing👍.
- [x] Flexbox has been used to layout elements👍.


 ## Required Changes ♻️

- [ ] From your commit message and while I checked the project, I noticed the only change you implemented is laying out your form using `flexbox`😉. Kindly do well to implement the other changes as shown in the previous review (Button interaction, spacings around elements etc)😉.
image


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -174,95 +182,98 @@ <h3>
...
+          <textarea
+            name="message"
+            id="message"
+            cols="30"
+            rows="10"
+            placeholder="Write your message here..."
+            maxlength="500"
+            required
+          ></textarea>
+          <input type="submit" value="Get in touch" />
```
Comment (shahadat3669) [inline index.html]: - [ ] 👍 Great job on implementing styles for the default and hover stages of the button! However, please ensure that you also include styles for the **pressed stage** to complete the button's visual behavior as specified in the design. By adding the active stage styles, you will ensure a consistent and interactive user experience. Keep up the good work! 💪🎉

image

File: index.html
Comment (shahadat3669) [inline index.html:1]: - [ ]  [Optional] 👍 It's important to make your contact form section responsive. Currently, the webpage seems to have a fixed size suitable for a 375px device width, but it's essential to make it responsive across different screen sizes. Please ensure that the form adapts and looks good from 375px to a minimum desktop screen size of 768px.

[Portfolio (3).webm](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/55840999/6025aa33-a3ce-41e0-8d24-fe40915bc0c4)

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi Everyone, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
✅ Good PR Title
✅ Good PR Description
✅ No linter errors 👍 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @Drew246 , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Great design
✅ ... Green linters

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**